FormValidator/index.html

```diff
@@ -80,14 +81,13 @@ <h1> Form Validator</h1>
         const data = new FormData(this);
         for (const entry of data) {
             if(entry[1].length === 0) {
-                console.log(entry[0])
                 $result.innerText = 'All field should not empty';
```

Comment:
Suggestion: move the text to the separate file for easy maintenance of the content